mobile/src/main/java/ch/epfl/sdp/mobile/ui/setting/EditProfileNameDialog.kt

```diff
@@ -0,0 +1,44 @@
+package ch.epfl.sdp.mobile.ui.setting
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+import ch.epfl.sdp.mobile.ui.prepare_game.Dialog
+
+/**
+ * Component for the Setting profile screen
```

Comment:
You may want to specify that this is a dialog that lets the user edit their profile name ^^

Reply:
Ah, true -> is changed